src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.ServiceClientAdapter/BMSAPIs/ContainerAPIs.cs

```diff
@@ -23,6 +23,22 @@ namespace Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.ServiceClient
 {
     public partial class ServiceClientAdapter
     {
+
+        public RestAzureNS.AzureOperationResponse InquireContainers(
+            string containerName,
+            ODataQuery<BMSContainersInquiryQueryObject> queryFilter,
```

Comment:
Where is `BMSContainersInquiryQueryObject `defined?

Reply:
Here : https://github.com/MabOneSdk/azure-powershell/blob/26b44278a10809fd9e19447060d7e44e22d27a5c/src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Backup.Providers/Providers/AzureFilesPsBackupProvider.cs#L330